rwcore/CMakeLists.txt

```diff
@@ -66,7 +66,10 @@ target_link_libraries(rwcore
         OpenGL::GL
     )
 
-openrw_target_apply_options(TARGET rwcore)
+openrw_target_apply_options(TARGET rwcore
+    COVERAGE
+    COVERAGE_EXCEPT gl/gl_core_3_3.c gl/gl_core_3_3.h
```

Comment:
Maybe also tests?

Reply:
That file is gigantic and full of functions we don't use.

Reply:
I thought about directory `tests`. Now it's treated as normal code and wrongly increase (overall) coverage.

Reply:
I'm sorry. I misread your comment.
There is something to be told about adding/removing unit tests.

I am for including the unit tests because:
- currently, coverage is low in the unit tests because we lack game data on ci
- this is related to the `TESTS_NODATA` compile flag (that should disappear)
- coverage is _just a number_